formbricks/formbricks
# Feature/pin protected surveys

## What does this PR do?
This PR implements the feature to PIN protect the sureys

Fixes #1089

![Screenshot 2023-10-14 at 12 23 05 PM](https://github.com/formbricks/formbricks/assets/44976328/9927a533-30dc-4324-b9a4-8f0bb7e7d4e4)


## Type of change

## How should this be tested?

Create a new Survey and add a PIN to the survey.
Now, If you's try to submit the Survey, you'll see an input to input your PIN

## Checklist



### Required


### Appreciated
